src/controllers/IllnessStartDateController.ts

```diff
@@ -0,0 +1,67 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+
+import { BaseController } from 'app/controllers/BaseController';
+import { DateValidator } from 'app/controllers/validators/DateValidator';
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { Reasons } from 'app/models/Reasons';
+import { Feature } from 'app/utils/Feature';
+import { ILLNESS_START_DATE_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template: string = 'illness/illness-start-date';
+
+const navigation: Navigation = {
+    previous(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    },
+    next(): string {
+        return ILLNESS_START_DATE_PAGE_URI;
+    }
+};
+
+interface FormBody {
+    illnessStart: string;
```

Comment:
This does not match actual values in `request.body`. It should have `date: Date` there.